### ISSUE TITLE
Excuse me, which images are to be trained for --image and --mask_path?

### ISSUE THREAD

**Issue by zys1999** [Qsingle/LearnablePromptSAM#17]
Hello, author. I am trying to train the AROI dataset using your model. I set the paths for the original images and label images as --image AROIonline/inter_intra/raw and --mask_path AROIonline/inter_intra/groundTruth, respectively. I trained for 32 epochs with a learning rate of 0.05, but the IoU was only 0.016 after training. Could you please advise on how to correctly set the paths for the original images and label images? Thank you.

**Comment by zys1999** [comment]
Additionally, I have not processed the AROI dataset, and I set numClass to 256. What other steps are necessary to achieve approximately 40% as mentioned in your paper?

**Comment by Qsingle** [comment]
Thank you for your question. You can set the `--mask_path` for the AROI dataset to `patient/mask/number`. For example, `patient12/mask/number` and set the `--image` to `patient12/raw/labelled`.

**Comment by zys1999** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by zys1999** [comment]
Thank you very much for your guidance. However, after setting up the --image and --mask_path as per your instructions, the IoU after training is still only 3%. My command line parameters are as follows:python train_learnable_sam.py --image data/...  --mask_path data/... --checkpoint sam_vit_h_4b8939.pth --lr 0.05 --num_class=256，All other settings are kept as default. Could you please advise on how to properly configure the command line parameters to achieve the accuracy mentioned in the paper? Thank you.

**Comment by zys1999** [comment]
> Thank you for your question. You can set the `--mask_path` for the AROI dataset to `patient/mask/number`. For example, `patient12/mask/number` and set the `--image` to `patient12/raw/labelled`.

Thank you very much for your guidance. However, after setting up the --image and --mask_path as per your instructions, the IoU after training is still only 3%. My command line parameters are as follows:python train_learnable_sam.py --image data/...  --mask_path data/... --checkpoint sam_vit_h_4b8939.pth --lr 0.05 --num_class=256，All other settings are kept as default. Could you please advise on how to properly configure the command line parameters to achieve the accuracy mentioned in the paper? Thank you.

**Comment by zys1999** [comment]
> Thank you for your question. You can set the `--mask_path` for the AROI dataset to `patient/mask/number`. For example, `patient12/mask/number` and set the `--image` to `patient12/raw/labelled`.

Thank you very much for your guidance. However, after setting up the --image and --mask_path as per your instructions, the iou after training is still only 3%. My command line parameters are as follows:python train_learnable_sam.py --image data/...  --mask_path data/... --checkpoint sam_vit_h_4b8939.pth --lr 0.05 --num_class=256，All other settings are kept as default. Could you please advise on how to properly configure the command line parameters to achieve the accuracy mentioned in the paper? Thank you.
